### ISSUE TITLE
"Create New Project" cannot handle nested modules

### ISSUE THREAD

**Issue by asarkar** [rikvdkleij/intellij-haskell#246]
Installed Stack in `/usr/local/bin/stack`, then ran `stack install hindent`, almost fell asleep, woke up and ran `stack install stylish-haskell`. No such option to "Import project from external model", tried "Create New Project", selected Stack binary on 1st window, filled in project location, name on 2nd, clicked Finish. Boom!

> Can not start project because Stack/Cabal info can not be retrieved.

Hello? I selected Stack binary to begin with.

I found https://github.com/rikvdkleij/intellij-haskell/issues/208 that is eerily similar, and this:

```
The file '/Users/asarkar/Workspace/haskell/learn-haskell/src' is not under content entry root '/Users/asarkar/Workspace/haskell/learn-haskell/recursion'
java.lang.Throwable: The file '/Users/asarkar/Workspace/haskell/learn-haskell/src' is not under content entry root '/Users/asarkar/Workspace/haskell/learn-haskell/recursion'
	at com.intellij.openapi.diagnostic.Logger.error(Logger.java:136)
	at com.intellij.openapi.roots.impl.ContentEntryImpl.assertFolderUnderMe(ContentEntryImpl.java:378)
...
```

The plugin did create a project that it ironically choked on, which looks like the following:

<img width="191" alt="haskell" src="https://user-images.githubusercontent.com/1302775/34196294-5918ffc2-e516-11e7-81fc-3881b17f5615.png">

**Comment by asarkar** [comment]
This appears to happen when the module is nested under the project. I'm just beginning to learn Haskell, but the names suggest that a project may contain multiple modules, so nesting should work. In the screenshot above, the project name is `learn-haskell` and module name `recursion`. See screenshot below for a similar setup:

<img width="756" alt="haskell" src="https://user-images.githubusercontent.com/1302775/34196609-bb2b241e-e517-11e7-9e10-696874e0f5b8.png">

**Comment by rikvdkleij** [comment]
The Stack template that is used when creating new project does not contain multiple modules, just one project which is one IntelliJ module. 

The project location has to be the same as module his Content-root and Module file location.

**Comment by asarkar** [comment]
This should be mentioned clearly in the doc.

**Comment by Sir4ur0n** [comment]
Can we close this?

**Comment by asarkar** [comment]
@Sir4ur0n If the problem is resolved, sure, why not? But I'm not seeing any comment saying that it has been resolved, and if it, what's the behavior now. I see a related commit but the commit message ("Fix issue #246") that doesn't help much.

**Comment by rikvdkleij** [comment]
Yes, it is solved in a way that `More Settings` is not available anymore.
